apisix/plugins/server-info.lua

```diff
@@ -99,6 +99,17 @@ local function get()
 end
 
 
+local function get_server_info()
+    local info, err = get()
```

Comment:
Look like we need to way to store the `boot_time` in shdict when `etcd` is not used. Otherwise the `boot_time` will change when plugin reloads.

Reply:
That's really, i will change it.

Reply:
Fixed.